Assets/AnkrSDK/Examples/UseCases/WebGlLogin/WebGLConnectionController.cs

```diff
@@ -6,55 +6,80 @@ namespace AnkrSDK.Examples.UseCases.WebGlLogin
 {
 	public class WebGLConnectionController : MonoBehaviour
 	{
-		[SerializeField]
-		private WebGLConnect _webGlConnect;
-
 		[SerializeField]
 		private WebGLLoginPanelController _webGlLoginManager;
-		
+
 		[SerializeField]
 		private GameObject _sceneChooser;
 
+		
+		private WebGLConnect WebGLConnect
```

Comment:
Private field name starts with a capital liter. Is it according to our code style?

Reply:
This is a private property, not a field